_scripts/release_component.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+NEW_RELEASE="${NEW_RELEASE:-unknown}"
+DRY_RUN_CMD='echo'
+DRY_RUN=YES
+UPDATE_MASTER=YES
+
+# flags are --help --no-dry-run --no-update-master
+for i in "$@"; do
+    case $i in
+        -h|--help)
+        echo "Usage: release_component.sh [--no-dry-run] [--no-update-master]"
```

Comment:
I always like indenting the contents of the case statement - makes it easier to skim through vertically.